update langauge (#5698)
flyteorg/flyte: docs/quickstart_guide.md

````{dropdown} Try Flyte in your browser
:animate: fade-in-slide-down

> Union Serverless is a version of Flyte hosted by [Union](https://www.union.ai/) that you can try in your browser. Sign up below and check out the [Union Serverless Quickstart guide](https://docs.union.ai/serverless/quick-start).

Reviewer: ```suggestion
Union Serverless is a SaaS offering by [Union](https://www.union.ai/) built on Flyte. Sign up below and check out the [Union Serverless Quickstart guide](https://docs.union.ai/serverless/quick-start).
```

wdyt?
